README.md

```diff
@@ -473,6 +473,35 @@ REALM_GRAPHQL_API_KEY=xxxxxxxxxx
 ```
 About Realm API Keys: https://www.mongodb.com/docs/realm/authentication/api-key/
 
+## Facebook login integration
```

Comment:
Looks like the title should mention Google too, or the more generic term "social login"